Content.Client/Medical/CrewMonitoring/CrewMonitoringWindow.xaml.cs

```diff
@@ -15,6 +15,11 @@ public sealed partial class CrewMonitoringWindow : DefaultWindow
     {
         private List<Control> _rowsContent = new();
 
+        // Limiter for how long a mobs's name or job can be. Without this the HoP can set a ridiculously
+        // long name and make it near impossible for someone to use the crew monitoring window.
+        private int _nameMaxLength = 30;
+        private int _jobMaxLength = 30;
```

Comment:
this should really be enforced in the ID card console system, rather than just visually here. fixing it there would also fix a number of other visual bugs that occur from super long ID names

Reply:
I was trying to limit the scope, but sure I'll look up the id card console and set the limit there.

Reply:
> this should really be enforced in the ID card console system, rather than just visually here. fixing it there would also fix a number of other visual bugs that occur from super long ID names

We already have limits, they just need reducing.